### ISSUE TITLE
Source archives should not package `.git`

### ISSUE THREAD

**Issue by mathstuf** [vrpn/vrpn#291]
I see the archive uploaded for the releases contains the `.git` directory. Instead of (presumably) packing a source tree, I would recommend using [`git-archive-all`](https://github.com/roehling/git-archive-all) instead to avoid packing the entire history too.

**Comment by rpavlik** [comment]
Ah, yeah, git-archive-all is quite useful for projects with submodules. If it existed at the time of the last release, at least I didn't know about it, and if it's a weird git tool I don't know about, I'm sure Russ didn't know about it :sweat_smile:

**Comment by mathstuf** [comment]
Yeah, it's a Python script that's been around for years. It's not perfect, but the corner cases are highly unlikely to matter in most cases.

**Comment by mathstuf** [comment]
Oh, seems it got ported to bash at some point? Or my memory is confusing it with some other tool.
